README.md

```diff
@@ -17,83 +21,75 @@ Run from the root directory
+```bash
+export ARCADE_API_KEY=test
+```
+
+3) Run Go Clouddriver.
 ```bash
 make run
 ```
-You should see a log like `[CLOUDDRIVER] DB_HOST, DB_NAME, DB_PASS, or DB_USER not defined; defaulting to local SQLite DB` - this is expected when running locally. For production, you should set the env variables `DB_HOST`, `DB_NAME`, `DB_PASS`, and `DB_USER`.
 
-3) Create your first Kubernetes provider! go-clouddriver runs on port 7002, so you'll make a POST to `localhost:7002/v1/kubernetes/providers`.
+4) Create your first Kubernetes provider! Go Clouddriver runs on port 7002, so you'll make a POST to `localhost:7002/v1/kubernetes/providers`.
 ```bash
 curl -XPOST localhost:7002/v1/kubernetes/providers -d '{
   "name": "test-provider",
   "host": "https://test-host",
   "caData": "test",
   "permissions": {
     "read": [
-      "test-read-group"
+      "test-group"
     ],
     "write": [
-      "test-write-group"
+      "test-group"
     ]
   }
 }' | jq
 ```
-And you should see the response...
+And you should see the response
 ```json
 {
   "name": "test-provider",
   "host": "https://test-host",
   "caData": "test",
   "permissions": {
     "read": [
-      "test-read-group"
+      "test-group"
     ],
     "write": [
-      "test-write-group"
+      "test-group"
     ]
   }
 }
 ```
 Running the command again will return a `409 Conflict` unless you change the name of the provider.
 
-4) List your providers by calling the `/credentials` endpoint.
+5) List your providers by calling the `/credentials` endpoint.
 ```bash
 curl localhost:7002/credentials | jq
 ```
 
-### Verbose Request Logging
-
-Building go-clouddriver requires a lot of reverse engineering and monitoring incoming requests.
-
-Turn on verbose request logging by setting the environment variable `VERBOSE_REQUEST_LOGGING` to `true`. You'll now see helpful request logs.
-
-```
-REQUEST: [2020-09-17T14:26:00Z]
-POST /v1/kubernetes/providers HTTP/1.1
-Host: localhost:7002
-Accept: */*
-User-Agent: curl/7.54.0
-{
-  "name": "test-provider",
-  "host": "https://test-host",
-  "caData": "test",
-  "permissions": {
-    "read": [
-      "test-read-group"
-    ],
-    "write": [
-      "test-write-group"
-    ]
-  }
-}
-
-[GIN] 2020/09/17 - 10:24:18 | 201 |     5.19472ms |       127.0.0.1 | POST     "/v1/kubernetes/providers"
-```
+### Configuration
+
+| Environment Variable | Description | Notes | Default Value |
+|----------|:-------------:|-----------:|------------:|
+| `ARCADE_API_KEY` | Needed to talk to [Arcade](https://github.com/billiford/arcade). | Required for most operations. ||
+| `ARTIFACTS_CREDENTIALS_CONFIG_DIR` | Sets the directory for artifacts configuration. | Optional. Leave unset to use OSS Clouddriver's Artifacts API. ||
+| `KUBERNETES_USE_DISK_CACHE` | Stores Kubernetes API discovery on disk instead of in-memory. || `false` |
+| `DB_HOST` | Used to connect to MySQL datastore. | If not set will default to local SQLite DB. ||
```

Comment:
Descriptions for MySQL sound more like the default :)

Reply:
Can we refer to the DB as MySQL 'database' everywhere instead of 'datastore'?